src/bindings/js/node/src/infer_request.cpp

```diff
@@ -215,7 +221,13 @@ void performInferenceThread(TsfnContext* context) {
             const auto& tensor = context->_ir->get_tensor(node);
             auto new_tensor = ov::Tensor(tensor.get_element_type(), tensor.get_shape());
             tensor.copy_to(new_tensor);
-            outputs.insert({node.get_any_name(), new_tensor});
+            std::string name;
```

Comment:
On the line 217 we get compiled model outputs and assign it to variable compiled model. This isn't good name. Please refactor it little bit.

Reply:
Ok, done